CONTRIBUTING.md

```diff
@@ -69,9 +69,9 @@ git merge upstream/main
 
```

Comment:
Could you please revert the changes in that file 🙏

Reply:
@Cahllagerfeld , reverted `CONTRIBUTING.md` to `29f3726f70a3815687d5d4b84d6880945aef34be`